docs/reference/settings/health-api-settings.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[health-api-settings]]
+=== Health API settings in {es}
+[subs="attributes"]
+++++
+<titleabbrev>Health API settings</titleabbrev>
+++++
+
+The following are the _expert-level_ settings available for configuring the
+<<health-api, Health API>>. It is not recommended to change any of these
+from their default values.
+
+==== Cluster level settings
+
+`health.master_history.has_master_lookup_timeframe`::
+(<<static-cluster-setting,Static>>) The amount of time the API looks back to see if the cluster has had
+a master at all, before moving on with other checks. Defaults to `30s` (30 seconds).
+
+`master_history.max_age`::
+(<<static-cluster-setting,Static>>) The maximum amount of time that the master history is kept in memory
+for use in the Health API. Master node changes older than this time will not be considered in the Health
```

Comment:
```suggestion
. Master node changes older than this time will not be considered in the Health
```

Reply:
I think we should potentially decouple these settings from the health API (I see it's mentioned again)

Reply:
If we don't say how these impact the end user, then no one would ever change them, right? That might not be such a bad thing though.